### PR TITLE
Products filter feature.

### PR DESCRIPTION
**Pull request by samyukthasudhakar**
This PR contains product filter functionality on the product listing page with the following features :

- Sort by Price: A radio button to sort from low to high & high to low.
- Category: Checkboxes with various categories according to the theme.
- Type: Checkboxes with various product types according to the theme.
- Ratings: A slider for ratings.
- Clear Button - To clear all the applied filters.

Files to review are part of this [commit ](https://github.com/samyukthasudhakar/paws-and-beyond-React/pull/5/commits/3cb4d9efd93d6a7924872148464f012bfccae759) .

Live Link : https://ecommerce-product-filter.netlify.app/

### CODE REVIEW

File: src/components/filterSideBar/filterSideBar.jsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import {useProducts} from '../../context/productsContext'
+
+function FilterSideBar(){
+
+    const { data:{sortByPrice, categoryFilters, typeFilters, ratingFilter}, productsDispatch } = useProducts()
+    const categories = ['Dogs', 'Cats', 'Birds', 'Fishes', 'Bunnies']
+    const productType = ['Accessory', 'Food', 'Medicine', 'Treat', 'Toy']
```
Comment (mikesch803) [inline src/components/filterSideBar/filterSideBar.jsx:7]: suggestion : you can separate out these arrays from this file and put in the data folder

**Review by mikesch803 — APPROVED** [review]
just a small suggestion that i put else your code is good to merge

**Comment by samyukthasudhakar** [comment]
> just a small suggestion that i put else your code is good to merge

Thank you 😄

**Review by jalajcodes — APPROVED** [review]
Code looks good. I likes your UI. Good work all in all.

Just one suggestion, you should format your code. I noticed you are not using prettier. It will make your life so much easier, you can set vscode to automatically format your code whenever you save the file.

**Comment by samyukthasudhakar** [comment]
> Code looks good. I likes your UI. Good work all in all.
> 
> Just one suggestion, you should format your code. I noticed you are not using prettier. It will make your life so much easier, you can set vscode to automatically format your code whenever you save the file.

Thanks for the feedback. I am using prettier, not sure what happened. Will do take a look at this.

**Pull request merged**